components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json

```diff
@@ -196,6 +196,22 @@
       {
         "Name": "apim:admin_operations",
         "Roles": "admin"
+      },
+      {
+        "Name": "registry:view",
+        "Roles": "admin,Internal/registry_admin,Internal/registry_user,Internal/registry_browser"
```

Comment:
Shall we add Internal/creator,Internal/publisher roles as well? Those users should be able to view the registries created because it will be used in the publisher portal.

Reply:
fixed